src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -1953,10 +1953,6 @@
 
 				self._unbindEvents();
 
-				self._items.forEach(function (li) {
-					self.element.appendChild(li);
-					li.setAttribute("style", "");
-				});
```

Comment:
@TomaszLukawskiSam, @lslachciak ,
Do you know why appending items on the destroy?

Reply:
When the widget are building the items are moved from element to widget cache (dummy element) and next appended to carousel. On destroy method these elements should be recovered to previous structure.

Reply:
@TomaszLukawskiSam,
I know what you mean.
However, the items added through the virtual list are also restored to the element.
So it makes some issue #1249.

Reply:
Ok. I see this now. This make sense to remove this code but we need create issue to recovery html structure in future.